app/models/author.rb

```diff
@@ -0,0 +1,10 @@
+class Author < ApplicationRecord
+  has_many :posts, dependent: :destroy
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
+
+  # A method that returns the 3 most recent posts for a given user.
+  def recent_posts
+    posts.order(created_at: :desc).limit(3)
+  end
+end
```

Comment:
based on the requirement you ought to have a User model, not an Authors model setup. 😄

Reply:
Hey @alexander16108 ,
Thank you for the review. Sure, I'm going to fix that. ✋